lib/msal-angular/src/packageMetadata.ts

```diff
@@ -1,3 +1,3 @@
 /* eslint-disable header/header */
-export const name = '@azure/msal-angular';
-export const version = '3.0.7';
+export const name = "@azure/msal-angular";
+export const version = "3.0.8";
```

Comment:
```suggestion
export const name = '@azure/msal-angular';
export const version = '3.0.8';
```

For some reason es-lint complains about this for angular only

Reply:
Good to know - we might consider adding the format:fix command after the bump to resolve